cdp_scrapers/instances/portland.py

```diff
@@ -100,50 +126,127 @@ def make_efile_url(efile_page_url: str) -> str:
     return f"{efile_page_url}File/Document"
 
 
+def get_disposition(minute_section: Tag) -> str:
+    """
+    Return disposition string given within minute_section <div>
+    on the event web page
+
+    Parameters
+    ----------
+    minute_section: Tag
+        <div> within event web page for a given event minute item
+
+    Returns
+    -------
+    disposition: str
+        Disposition string for the event minute item
+        e.g. Accepted, Passed, Placed on file
+    """
+    result_status_element_sibling = minute_section.find(
+        "div", text=re.compile("Disposition"), attrs={"class": "field__label"}
+    )
+    result_status_element = result_status_element_sibling.next_sibling
+    return result_status_element.text
+
+
+def disposition_to_minute_decision(
+    disposition: str,
+) -> Optional[EventMinutesItemDecision]:
+    """
+    Decide EventMinutesItemDecision constant from event minute item disposition
+
+    Parameters
+    ----------
+    disposition: str
+        Disposition event web page for a given item
+        e.g. Passed, Continued
+
+    Returns
+    -------
+    decision: Optional[EventMinutesItemDecision]
+
+    See Also
+    --------
+    MINUTE_ITEM_PASSED_PATTERNS
+    """
+    for pattern in MINUTE_ITEM_PASSED_PATTERNS:
+        if re.search(pattern, disposition, re.I):
+            return EventMinutesItemDecision.PASSED
+    return None
+
+
+def separate_name_from_title(title_and_name: str) -> str:
+    """
+    Return just name
+
+    Parameters
+    ----------
+    title_and_name: str
+        e.g. Mayor Ted Wheeler
+
+    Returns
+    -------
+    name: str
+        tile_name_name with first word removed e.g. Ted Wheeler
+
+    Notes
+    -----
+    first word in title_and_name is presumed to be title
+    """
+    # title_and_name:
+    #   The title (Mayor of Commissioner) and name of a Portland City Commission
+    #   member.
+    #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio
+
+    name_index = title_and_name.find(" ")
```

Comment:
Is there ever a case where the title has a space in it? Like "Vice President"

Reply:
Ooh great point.. For Portland specifically I don't believe so. I will definitely double check. I _think_ I've only seen either Mayor, or everyone else as Commissioner. Yeah, the terminology is quite different from e.g. Seattle. 

Nonetheless I'm going to make sure on this. It'll obviously break the logic.